backend/src/controllers/user.js

```diff
@@ -1,82 +1,123 @@
-// Dummy data
-const users = [
-    {
-        id: 1,
-        name: 'Alex'
-    },
-    {
-        id: 2,
-        name: 'Shawn'
-    },
-    {
-        id: 3,
-        name: 'Craig'
-    },
-    {
-        id: 4,
-        name: 'Adam'
-    }
-]
+const User = require("../models/user.js");
+const what = { single: 'user', plural: 'users' };
```

Comment:
I'm guessing this is a step toward having a standardized boilerplate for responses?

Reply:
when i see repeating code, i break stuff out like this. it helps for code re-use. when you create new endpoints, you only have to duplicate the file and change a string or two to start off your new development. honestly, i should have called `User` `Model` instead, to make it even more generic. the filename is already telling us that this is user stuff.